test/smoke/cluster_master_sites_response.go

```diff
@@ -15,7 +15,7 @@
 package smoke
 
 // ClusterMasterSitesResponse is a representation of the sites managed by a Splunk cluster-manager
-// Endpoint: /services/cluster/master/sites
```

Comment:
Some endpoints have been migrated to use manager as 
[here](https://docs.splunk.com/Documentation/Splunk/8.2.2/RESTREF/RESTcluster#cluster.2Fmanager.2Fsites). But not all. 

Our general approach was to migrate all comments while using constants for literals (non-comments).